moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -409,18 +412,30 @@ bool IKFastKinematicsPlugin::initialize(const moveit::core::RobotModel& robot_mo
   }
 
   storeValues(robot_model, group_name, base_frame, tip_frames, search_discretization);
+  if (!lookupParam("linkprefix", linkframeprefix_, std::string("")))
+  {
+    ROS_INFO_NAMED(name_, "linkprefix parameter not set. Assuming empty prefix.");
```

Comment:
If the `else` case here is a `DEBUG` log, I would expect this to be the same.

Reply:
For conistiency making both ROS_INFO, see below https://github.com/ros-planning/moveit/pull/1599#discussion_r317199820

Reply:
```suggestion
    ROS_INFO_NAMED(name_, "link_prefix parameter not set. Assuming empty prefix.");
```